mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,14 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a,b,c) {
+    return Number = a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe( name, age ){
+    return "Hello,my name is " + name + "and I am " + age + "years old";
+}
 function getTotal(a, b) {
```

Comment:
This test failed as you returned the total as just an integer rather than the integer in a message string